### DISCUSSION TITLE
Anaysis Falling Behind - Recovery?

### DISCUSSION THREAD

**Discussion by S-Tuthill** [mcguirepr89/BirdNET-Pi#916] [General]
I have a new BirdNET system running based on a Libre Le Potato. I noticed this morning that the analysis fell behind and a message as such showed up on the Overview page. Something like "hours to catch up".  I have never seen this on my Raspberry Pi system. My question is " Is there a command I can issue that tells BirdNET to give up on the backlogged audio and start fresh?". I tried a Restart but that did not do it.

Thanks for any ideas.

**Comment by blogdemoi** [comment]
I also have a Le Potato and at the beginning of using it I was also behind.

I think you should change your settings for:

-Recording length=30
-Extraction length=6

And let it quietly catch up and keep those settings after.

I am running with those settings and now it is rarely behind.

**Reply by blogdemoi** [reply]
I also have a 500Mib swap size and a case with a fan.
